lib/screens/home_page.dart

```diff
@@ -15,7 +16,12 @@ class _HomePageState extends State<HomePage> {
     return Scaffold(
       appBar: HomePageAppBar(),
       drawer: CustomDrawer(),
-      body: MapWidget(),
+      body: Stack(
```

Comment:
I think the stack is not needed here. Can you please verify?

Reply:
I'm not sure what I should replace it with.

Reply:
I reviewed the drawer you have implemented and indeed it requires a Stack. Thanks